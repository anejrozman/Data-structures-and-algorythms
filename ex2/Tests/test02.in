10,40,0
67.55,35.62,85.55,64.62
9.9,70.37,26.9,93.37
60.33,5.03,80.33,27.03
40.34,37.92,55.34,49.92
69.24,84,92.24,100
41.49,0,46.49,13
50.83,69.54,60.83,91.54
9.58,28.64,16.58,47.64
25.3,26.31,34.3,39.31
34.46,54.93,41.46,81.93

67.55,35.62
67.55,64.62
85.55,35.62
85.55,64.62
9.9,70.37
9.9,93.37
26.9,70.37
26.9,93.37
60.33,5.03
60.33,27.03
80.33,5.03
80.33,27.03
40.34,37.92
40.34,49.92
55.34,37.92
55.34,49.92
69.24,84
69.24,100
92.24,84
92.24,100
41.49,0
41.49,13
46.49,0
46.49,13
50.83,69.54
50.83,91.54
60.83,69.54
60.83,91.54
9.58,28.64
9.58,47.64
16.58,28.64
16.58,47.64
25.3,26.31
25.3,39.31
34.3,26.31
34.3,39.31
34.46,54.93
34.46,81.93
41.46,54.93
41.46,81.93
