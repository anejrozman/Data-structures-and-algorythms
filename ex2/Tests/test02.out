154.53623051382124
145.25071441415378
158.73294335294622
152.29772322190308
182.93551295464624
196.95032146059492
170.9526774744146
185.1370838959919
173.89695795660813
153.5556648513458
181.43537952119132
162.14187776343385
148.18729998069423
145.0253143194788
145.61482253218514
147.36495613902514
150.90834096225365
181.7802075879341
145.02531431947884
151.00280605995596
161.52964314599393
150.5186047251301
178.61425194009115
153.39982177486164
148.94063292252895
169.09068825424572
145.7591305567734
167.47564720670547
149.25714319948605
157.5912368051899
145.15036518275448
154.31368189501856
147.77818338649078
145.0253143194788
145.78870468612408
147.82826322561837
148.1390741778471
170.36254513752323
145.51766870041666
171.72258304081117
